site/_plugins/algolia_hooks.rb

```diff
@@ -8,12 +8,32 @@ def self.should_be_excluded?(filepath)
       end
 
       def self.before_indexing_each(record, _node, _context)
+        if _node != nil
+
+          # Simplify h1, h2, h3, h4, h5 and h6 indexing in case of embedded elements.
+          # - Specifically, the h1 in release notes includes an inner <a> that does not need to be indexed.
+          if %w[h1 h2 h3 h4 h5 h6].include? _node.name
+            record[:html] = _node.text.strip
```

Comment:
We delete this value on line 40, probably not worth doing.

Reply:
Good point. Thanks, I saw that too.. I'm an idiot.